src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -23,7 +23,10 @@
 @Component
 public class TrustDetailsValidator {
 
-    public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+    public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+                           Errors errors,
+                           String loggingContext,
+                           boolean checkCeasedDate) {
```

Comment:
Method is called with the boolean variable isFullValidation mapped to checkCeasedDate. Might be more readable if isFullValidation is used throughout.

Reply:
Yeah. I can do that. Will change the parameter name.